.github/workflows/installcheck.yml

```diff
@@ -0,0 +1,52 @@
+name: PG12 Regression
```

Comment:
This workflow is duplicate. There are two same workflows `installcheck.yml` and `installcheck.yaml`.

Reply:
Should we delete the yml one?

Reply:
yes

Reply:
yes